mkdocs/docs/backends/gpu-vulkan.md

```diff
@@ -1,10 +1,149 @@
-# Vulkan GPU backend
+# Vulkan GPU HAL Driver
 
-<!-- TODO(??): overview, when to use CPU vs GPU -->
+IREE can accelerate model execution on GPUs via [Vulkan][vulkan], a low-overhead
+graphics and compute API. Vulkan is cross-platform: it is available on many
+operating systems, including Android, Linux, and Windows. Vulkan is also
+cross-vendor: it is supported by most GPU vendors, including AMD, ARM, Intel,
+NVIDIA, and Qualcomm.
+
+<!-- TODO(??): when to use CPU vs GPU -->
+
+<!-- TODO(??): support matrix of different GPUs -->
 
 ## Prerequisites
 
-<!-- TODO(??): build from source, vcpkg -->
+In order to use Vulkan to drive the GPU, you need to have a functional Vulkan
+environment. IREE requires Vulkan 1.1 on Android and 1.2 elsewhere. It can be
+verified by the following steps:
+
+=== "Android"
+
+    Android mandates Vulkan 1.1 support since Android 10. You just need to
+    make sure the device's Android version is 10 or higher.
+
+=== "Linux"
+
+    Run the following command in a shell:
+
+    ``` shell
+    vulkaninfo | grep apiVersion
+    ```
+
+    If `vulkaninfo` does not exist, you will need to [install the latest Vulkan
+    SDK][vulkan-sdk]. For Ubuntu 18.04/20.04, installing via LuarG's package
+    repository is recommended, as it places Vulkan libraries and tools under
+    system paths so it's easy to discover.
+
+    If the showed version is lower than Vulkan 1.2, you will need to update the
+    driver for your GPU.
+
+
+=== "Windows"
+
+    Run the following command in a shell:
+
+    ``` shell
+    vulkaninfo | grep apiVersion
+    ```
+
+    If `vulkaninfo` does not exist, you will need to [install the latest Vulkan
+    SDK][vulkan-sdk].
+
+    If the showed version is lower than Vulkan 1.2, you will need to update the
+    driver for your GPU.
+
+
+## Get Runtime and Compiler
```

Comment:
(bikeshedding on style for consistency)

I kept going back and forth between `Title Case` and `Only first word capitalized case`. I settled on using `Only first word capitalized case` for H1 (titles), H2, and H3.

Looks like you're using `Title Case` for H2 and `Only first word capitalized case` for H3/H4?

WDYT?

Reply:
I typically do "All Capital Case" for both H1 and H2 and use "Only first world capitalized case" for the rest because that's what typically books do (or at least the ones I read and can remember ;-P). But I'm fine either way and am in favor of consistency.  So changed.